Code/GraphMol/MolStandardize/MolStandardize.h

```diff
@@ -70,6 +70,12 @@ struct RDKIT_MOLSTANDARDIZE_EXPORT CleanupParameters {
   bool tautomerReassignStereo{
       true};  //! Whether enumerate() should call assignStereochemistry
               //! on all generated tautomers (defaults to true)
+  bool largestFragmentChooserUseAtomCount{
+      true};  //! Whether LargestFragmentChooser should use atom
+              //! count as criterion (defaults to true)
+  bool largestFragmentChooserHeavyOnly{
```

Comment:
consistent with the above suggestion:
```suggestion
  bool largestFragmentChooserCountHeavyAtomsOnly{
```